spring-integration-jmx/src/main/java/org/springframework/integration/monitor/IntegrationMBeanExporter.java

```diff
@@ -630,6 +630,18 @@ public long getSourceMessageCountLong(String name) {
 		return -1;
 	}
 
+	public int getChannelSendCount(String name) {
+		return (int) getChannelSendCountLong(name);
+	}
+
+	public long getChannelSendCountLong(String name) {
+		if (channelsByName.containsKey(name)) {
+			return channelsByName.get(name).getSendCountLong();
```

Comment:
So, `getSendErrorCount()` will we cover on demand?
Wouldn't it be better just expose `getChannelMetrics(String name)` ?